modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -1,111 +1,54 @@
 [[public-cloud-azure]]
-= SUSE Manager Server in Azure
+= {productname} with Azure
```

Comment:
@Loquacity  @jcayouette Can we expand macros in chapter titles (now?)?

I'm also not sure whether all this will work with Uyuni.  Probably in general, but I did not check all the details.

Reply:
I thought that was an old restriction. Better get @jcayouette 's advice on that.